package.json

```diff
@@ -35,6 +35,7 @@
     "@ember/render-modifiers": "^1.0.2",
     "@glimmer/component": "^1.0.0",
     "@glimmer/tracking": "^1.0.0",
+    "@kanselarij-vlaanderen/au-kaleidos-icons": "^1.2.3",
```

Comment:
Waarom bij dev dependencies ?

Reply:
De filosofie van [`dependencies vs devDependencies`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#dependencies) is dat je in `dependencies` enkel dingen zet die nodig zijn om je package te kunnen runnen. In `devDependencies` zet je dingen die je nodig hebt tijdens development. Gezien Ember (maar ook meeste andere moderne FE-frameworks) je dependencies "packagen" tot één grote bol JS (of CSS for the matter) heb je deze dependencies at runtime niet nodig. Je hebt ze enkel nodig tijdens development (om een build te kunnen maken). Daarom horen ze niet thuis in "dependencies".
Je verwarring komt misschien met andere packages die ook onder `dependencies` staan. Die horen daar ook niet thuis, maar wilde ik binnen de scope van deze PR niet aanraken.